.github/workflows/cli-core-test.yml

```diff
@@ -22,21 +25,14 @@ jobs:
           cache: 'npm'
       - name: Run tests
         run: npm test
-  sonarcloud:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          # Disabling shallow clone is recommended for improving relevancy of reporting
-          fetch-depth: 0
       - name: SonarCloud Scan
         uses: sonarsource/sonarcloud-github-action@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
   notify-complete-fail:
-    if: ${{ failure() || cancelled() }}
-    needs: [ test, sonarcloud ]
+    if: ${{ failure() && github.ref == 'refs/heads/main' && github.event_name != 'pull_request'  }}
```

Comment:
Shouldn't we also notify in case of a cancelled workflow?

Reply:
I felt If someone cancels a workflow run intentionally we don't need to notify in slack. Currently for travis also we are not doing this. Please let me know if you think `cancelled` is good to have I can add it back.

Reply:
That makes sense, if cancelled can only be triggered manually, there's no need to notify for it.